mindmeld/models/tagger_models.py

```diff
@@ -73,12 +73,13 @@ class TaggerModel(Model):
     CRF_TYPE = "crf"
     MEMM_TYPE = "memm"
     LSTM_TYPE = "lstm"
-    ALLOWED_CLASSIFIER_TYPES = [CRF_TYPE, MEMM_TYPE, LSTM_TYPE]
+    TORCH_CRF_TYPE = "torch-crf"
+    ALLOWED_CLASSIFIER_TYPES = [CRF_TYPE, MEMM_TYPE, LSTM_TYPE, TORCH_CRF_TYPE]
 
     # for default model scoring types
     ACCURACY_SCORING = "accuracy"
     SEQ_ACCURACY_SCORING = "seq_accuracy"
-    SEQUENCE_MODELS = ["crf"]
+    SEQUENCE_MODELS = ["crf", "torch-crf"]
 
```

Comment:
Once the old crf codebase is removed, do we intend to remove the "crf" model name or "torch-crf" ? 
Just a thought: I think because there will be a less chance that we create a different backend based crf in near future (i.e. after this PR is merged), it would be a good idea to retain the old name "crf" as this torch based crf is going to be the de facto crf implementation in MindMeld.

Reply:
Raised https://github.com/cisco/mindmeld/issues/416 for this

Reply:
I'd say just mention the issue number in the comment.

Reply:
Done